### ISSUE TITLE
v11 and MSEide

### ISSUE THREAD

**Issue by lainz** [bgrabitmap/bgrabitmap#108]
Hi, the version 10.9 works fine with MSEide. But the version 11 doesn't works, I get this error messages:

```
Free Pascal Compiler version 3.3.1-r43969 [2020/01/18] for i386
Copyright (c) 1993-2019 by Florian Klaempfl and others
Target OS: Win32 for i386
Compiling button.pas
Compiling C:\Users\Damian\Documents\GitHub\bgragui\bgraguicontrols\msegui\bgbutton.pas
Compiling C:\Users\Damian\Documents\GitHub\bgrabitmap\bgrabitmap\bgrabitmap.pas
Compiling C:\Users\Damian\Documents\GitHub\bgrabitmap\bgrabitmap\bgramseguibitmap.pas
Compiling C:\Users\Damian\Documents\GitHub\bgrabitmap\bgrabitmap\bgratext.pas
bgramsegui_text.inc(5,32) Error: Identifier idents no member "Font"
bgramsegui_text.inc(6,18) Error: Identifier idents no member "DrawString"
bgramsegui_text.inc(16,63) Error: Identifier idents no member "Font"
bgramsegui_text.inc(18,35) Error: Identifier idents no member "GetStringWidth"
bgramsegui_text.inc(19,35) Error: Identifier idents no member "Font"
bgramsegui_text.inc(20,18) Error: Identifier idents no member "DrawString"
bgramsegui_text.inc(30,60) Error: Identifier idents no member "GetStringWidth"
bgramsegui_text.inc(31,68) Error: Identifier idents no member "GetStringWidth"
bgramsegui_text.inc(34,55) Error: Identifier idents no member "Font"
bgramsegui_text.inc(35,68) Error: Identifier idents no member "Font"
bgramsegui_text.inc(37,32) Error: Identifier idents no member "Font"
bgramsegui_text.inc(38,18) Error: Identifier idents no member "DrawString"
bgramsegui_text.inc(45,33) Error: Identifier idents no member "GetStringWidth"
bgramsegui_text.inc(45,72) Error: Identifier idents no member "Font"
bgramsegui_text.inc(64,23) Error: Identifier idents no member "GetStringWidth"
bgramsegui_text.inc(78,18) Error: Identifier idents no member "FillRect"
bgratext.pas(272,15) Error: Identifier idents no member "Font"
bgratext.pas(273,15) Error: Identifier idents no member "Font"
bgratext.pas(274,15) Error: Identifier idents no member "Font"
bgratext.pas(275,15) Error: Identifier idents no member "Font"
bgratext.pas(495,3) Warning: Unreachable code
bgratext.pas(505,20) Error: Identifier idents no member "Font"
bgratext.pas(506,20) Error: Identifier idents no member "Font"
bgratext.pas(507,20) Error: Identifier idents no member "Font"
bgratext.pas(509,20) Error: Identifier idents no member "Font"
bgratext.pas(545,19) Error: Identifier idents no member "Font"
bgratext.pas(546,19) Error: Identifier idents no member "Font"
bgratext.pas(547,19) Error: Identifier idents no member "Font"
bgratext.pas(696,22) Error: Identifier idents no member "Font"
bgratext.pas(699,24) Error: Identifier idents no member "Font"
bgratext.pas(702,24) Error: Identifier idents no member "Font"
bgratext.pas(708,24) Error: Identifier idents no member "Font"
bgratext.pas(743,22) Error: Identifier idents no member "Font"
bgratext.pas(746,24) Error: Identifier idents no member "Font"
bgratext.pas(749,24) Error: Identifier idents no member "Font"
bgratext.pas(1102,12) Error: Identifier idents no member "Font"
bgratext.pas(1103,12) Error: Identifier idents no member "Font"
bgratext.pas(1104,12) Error: Identifier idents no member "Font"
bgratext.pas(1179,12) Error: Identifier idents no member "Font"
bgratext.pas(1180,12) Error: Identifier idents no member "Font"
bgratext.pas(1181,89) Error: Identifier idents no member "Font"
bgratext.pas(1182,20) Error: Identifier idents no member "Font"
bgratext.pas(1183,12) Error: Identifier idents no member "Font"
bgratext.pas(1817) Fatal: There were 42 errors compiling module, stopping
Fatal: Compilation aborted
```

**Comment by lainz** [comment]
For MSEide I'm using FPC trunk 3.3.1

**Comment by lainz** [comment]
I will try switching to release compiler for MSEide. Fred says that it works in release candidate of 3.2, so maybe something is wrong with 3.3.1.

**Comment by lainz** [comment]
Solved using FPC 3.0.4.
